tests/tests/swfs/avm2/stage3d_texture/Camera3D.as

```diff
@@ -0,0 +1,710 @@
+package
+{
+	import flash.geom.Matrix3D;
+	import flash.geom.Vector3D;
+	
+	/**
+	 *   A 3D camera using perspective projection
+	 *   @author Jackson Dunstan
+	 */
```

Comment:
Is this code under a compatible license?

Reply:
This comes from the zip file at https://www.jacksondunstan.com/articles/1998
Unfortunately, there doesn't seem to be an explicit license. However, we're only using this a test - it's not linked into any part of Ruffle, and it's not distributed as part of releases.

Reply:
I guess we'd need to ask him
https://www.jacksondunstan.com/articles/1655

Reply:
I'm fine with including it, but it'd be nice get an explicit license as we do distribute source on GH releases. Maybe we could toss the author an e-mail?